### ISSUE TITLE
Operation ID Verb check bug

### ISSUE THREAD

**Issue by jianghaolu** [Azure/azure-openapi-validator#143]
This line: https://github.com/Azure/azure-openapi-validator/blob/master/src/typescript/azure-openapi-validator/rules/PostOperationIdContainsUrlVerb.ts#L32

checks if the first segment contains a verb, instead it should be

```typescript
var operationparts = operationId.toLowerCase().split('_');
operationparts.pop();
if (operationparts.indexof(urlVerb) === -1)
{
….
}

```

**Comment by vladbarosan** [comment]
@jianghaolu not sure what you mean. I think the existing logic is correct. 

we want the second part of the operationId to be the verb ( so we search the verb in the "popped" string after splitting ).
